[BUG] Cluster manager bootstrap takes time causing intermittent failures in integration tests (o.o.c.ClusterHealthIT.testHealthOnClusterManagerFailover)

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Run o.o.cluster.ClusterHealthIT.testHealthOnMasterFailover and enable [index creation](https://github.com/dreamer-89/OpenSearch/blob/996d33adb22ac6a523962166f3677f8dc6ac9c1f/server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java#L352).
2. Reduce the [master node timeout ](https://github.com/dreamer-89/OpenSearch/blob/996d33adb22ac6a523962166f3677f8dc6ac9c1f/server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java#L377) to <10 seconds.
3. Run test multiple times. It fails with >90% when 1 second timeout is used.

**Expected behavior**
Master node boot up time should stay less than < 1 minut.

**Host/Environment (please complete the following information):**
 - OS: iOS

Failure in https://github.com/opensearch-project/OpenSearch/pull/1874#issuecomment-1011640866 looks the same

```
> Task :server:internalClusterTest

REPRODUCE WITH: ./gradlew ':server:internalClusterTest' --tests "org.opensearch.cluster.ClusterHealthIT.testHealthOnMasterFailover" -Dtests.seed=2391EC7752804595 -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=es-EC -Dtests.timezone=Etc/Greenwich -Druntime.java=17

org.opensearch.cluster.ClusterHealthIT > testHealthOnMasterFailover FAILED
    java.lang.AssertionError: expected same:<RED> was not:<GREEN>
        at __randomizedtesting.SeedInfo.seed([2391EC7752804595:BD7EC35A516A5973]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotSame(Assert.java:829)
        at org.junit.Assert.assertSame(Assert.java:772)
        at org.junit.Assert.assertSame(Assert.java:783)
        at org.opensearch.cluster.ClusterHealthIT.testHealthOnMasterFailover(ClusterHealthIT.java:393)
```

--- a/server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java
+++ b/server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java
@@ -377,7 +377,7 @@ public class ClusterHealthIT extends OpenSearchIntegTestCase {
                     .prepareHealth()
                     .setWaitForEvents(Priority.LANGUID)
                     .setWaitForGreenStatus()
-                    .setClusterManagerNodeTimeout(TimeValue.timeValueMinutes(2))
+                    .setClusterManagerNodeTimeout(TimeValue.timeValueMinutes(3))
                     .execute()
             );
             internalCluster().restartNode(internalCluster().getClusterManagerName(), InternalTestCluster.EMPTY_CALLBACK);

